src/components/events/EditSignup/DeleteSignup.tsx

```diff
@@ -29,7 +29,7 @@ const DeleteSignup = () => {
       toast.success("Ilmoittautumisesi poistettiin onnistuneesti.", {
         autoClose: 10000,
       })
-      navigate(paths.eventDetails(event!.slug))
+      navigate(paths.eventDetails(event?.slug ?? ""))
```

Comment:
Another option for this behaviour is to simply throw an error, but I think this will suffice for now.